README.md

```diff
@@ -46,6 +46,19 @@ gemrat --pessimistic cocoapods xcpretty gym deliver match
 
 ---
 
+## Upgrading CocoaPods in a legacy project
+
+If you have an older project that isn't using CocoaPods 1.0 and you want/need to update it, these steps should help:
+
+1. Make sure you're using a recent ruby version (2.3.x for now) - if you don't have it you can run `brew install ruby` or for a specific/multiple version(s) `brew install rvm`, `rvm install 2.3.1`, `rvm use 2.3.1`
+2. You'll need to set which new versions of gems/pods you want using `gem update [gemfile]` or `bundle exec update [podfile]`
```

Comment:
This should just refer to `bundle exec gem update`, `[podfile]` is not required.